src/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+# Uncomment for VERBOSE Makefiles. Useful for debugging.
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# project name and language
+project(ImpactT LANGUAGES Fortran)
+
+# BUILD_SHARED_LIBS is a global flag offered by CMake
+# to toggle the behavior of add_library
+# With OFF it will create a static library
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
The default in CMake is `off`, too.
No need to define the variable here, it's always present as a cache variable.

Reply:
Removed.